### ISSUE TITLE
[doc] 3.3.2.1  SUSE Linux Enterprise Server Installation

### ISSUE THREAD

**Issue by pablochacin** [SUSE/doc-caasp#491]
3.3.2.1  SUSE Linux Enterprise Server Installation

https://susedoc.github.io/doc-caasp/master/caasp-deployment/single-html/#_suse_linux_enterprise_server_installation

Instructions suggest using the "minimal" installation, but it doesn't include `sudo` which is needed.

**Comment by pablochacin** [comment]
Quick fix: add the following actions to section [3.3.2.2 Preparation of the VM as a Template](https://susedoc.github.io/doc-caasp/master/caasp-deployment/single-html/#_preparation_of_the_vm_as_a_template)

* change to root user: `su` 
* install sudo:  `zypper install sudo`
* add `sles` user to sudoers: `echo "sles ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers`
*  return to `sles` user: `exit`

**Comment by r0ckarong** [comment]
Installation of sudo is already part of the instructions though:

https://susedoc.github.io/doc-caasp/master/caasp-deployment/single-html/#_preparation_of_the_vm_as_a_template

Step 5:

```
Install required packages. As root, run:

zypper in sudo cloud-init cloud-init-vmware-guestinfo open-vm-tools
```
